node/src/main/scala/coop/rchain/node/api/WebApi.scala

```diff
@@ -308,6 +305,30 @@ object WebApi {
       ExprMap(fields.toMap).some
     } else none
 
+  private def parProtoFromExpr(exp: RhoExpr): Par = {
```

Comment:
This function should go just below `RhoExpr to protobuf` comment and should be named in the same way as others `exprToParProto`.